src/checkpatch_wrapper.sh

```diff
@@ -12,54 +20,56 @@ function execute_checkpatch()
 
   local -r options="--terse --no-tree --color=always -strict --file "
   local -r script="scripts/checkpatch.pl $options"
-  local -r original_working_dir=$PWD
+  local -r original_working_dir="$PWD"
   local kernel_root=""
 
-  if [[ -z $FILE_OR_DIR_CHECK ]]; then
+  if [[ -z "$FILE_OR_DIR_CHECK" ]]; then
     FILE_OR_DIR_CHECK="."
   fi
 
-  FILE_OR_DIR_CHECK="$(realpath $FILE_OR_DIR_CHECK)"
   # Check if is a valid path
-  if [[ ! -d $FILE_OR_DIR_CHECK && ! -f $FILE_OR_DIR_CHECK ]]; then
+  if [[ ! -d "$FILE_OR_DIR_CHECK" && ! -f "$FILE_OR_DIR_CHECK" ]]; then
     complain "Invalid path"
-    return 1
+    return 22 # EINVAL
   fi
 
-  # try to find kernel root at given path
+  # Get realpath for using inside checkpatch
+  FILE_OR_DIR_CHECK="$(realpath $FILE_OR_DIR_CHECK)"
+
+  # Try to find kernel root at given path
   kernel_root="$(find_kernel_root $FILE_OR_DIR_CHECK)"
   if [[ -z "$kernel_root" ]]; then
-    # fallback: try to find kernel root at working path
+    # Fallback: try to find kernel root at working path
     kernel_root="$(find_kernel_root $original_working_dir)"
   fi
 
-  # Check if kernel root was found.
+  # Check if kernel root was found
   if [[ -z "$kernel_root" ]]; then
     complain "Neither the given path nor the working path is in a kernel tree."
-    return 1
+    return 22 # EINVAL
   fi
+
   # Build a list of file to apply check patch
-  FLIST=`find $FILE_OR_DIR_CHECK -type f ! -name '*\.mod\.c' | grep "\.[ch]$" `
+  cmd="find $FILE_OR_DIR_CHECK -type f ! -name '*\.mod\.c' | grep \"\.[ch]$\""
+  FLIST=$(cmd_manager "SILENT" "$cmd")
```

Comment:
Since this is not a command we will present to the user nor use in tests, shouldn't we perhaps just invoke it directly? (without `cmd_manager`)